fix(snuba): Return error/sql again

wmak opened the pull request:
- Without debug in the query params snuba won't return the generated sql
- This way debug isn't in the body log too, making it easy to copy to send yourself

wmak committed:
fix(snuba): Return error/sql again

- Without debug in the query params snuba won't return the generated sql

Zylphrex commented on src/sentry/utils/snuba.py:
```diff
@@ -595,11 +595,11 @@ def snuba_query(params):
-                body = json.dumps(query_params)
                 referrer = headers.get("referer", "<unknown>")
                 if SNUBA_INFO:
+                    logger.info("{}.body: {}".format(referrer, query_params))
```
This no longer dumps valid JSON so we won't be able to easily copy + paste the snuba query any more. Is this fine?

wmak replied:
🤦 dangit you're right, included json.dumps in this format here, Wanted to keep the copy+paste ability but didn't want to introduce a second `if SNUBA_INFO` check

wmak committed:
fix: Adding json dumps to the logger

- I know this is double json.dumps, but this is for 2 reasons
  - Only one check of `SNUBA_INFO`
  - so we don't get `debug` in the json result
